WordPress/gutenberg
# [RNMobile] Added Audio block to the initial html

## Description

A section was created with the File and Audio block in the initial HTML that is shown when the editor loads in debug mode. 

## How has this been tested?

1. Run the Gutenberg demo app. 
2. Notice that the metro server sets the editor's content from the initial HTML. 
3. Observe that the File block is no longer at the top above the "Welcome to Gutenberg" title. 

Before | After 
--------|-------
 <kbd><img src="https://user-images.githubusercontent.com/1509205/108141599-0bc61880-7092-11eb-9bbd-f9e95ed8db6a.png" width="320"></kbd>  |      <kbd><img src="https://user-images.githubusercontent.com/1509205/108141601-0cf74580-7092-11eb-8945-2b16b80346a7.png" width="320"></kbd>
 
4. Observe that the File and Audio block is rendered in the Media Section

 <kbd><img src="https://user-images.githubusercontent.com/1509205/108141706-3912c680-7092-11eb-8559-e40160f97019.png" width="320"></kbd>





## Screenshots 

## Types of changes





## Checklist:
